streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/rpc/PbResultParser.java

```diff
@@ -0,0 +1,54 @@
+package io.ray.streaming.runtime.rpc;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import io.ray.streaming.runtime.generated.RemoteCall;
+import io.ray.streaming.runtime.message.CallResult;
+import io.ray.streaming.runtime.transfer.channel.ChannelRecoverInfo;
+import java.util.HashMap;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PbResultParser {
+
+  private static final Logger LOG = LoggerFactory.getLogger(PbResultParser.class);
+
+  public static Boolean parseBoolResult(byte[] result) {
```

Comment:
Ray support primitive types and string/binary serialization bwtween java/python. We shouldn't to repeat it again.

Reply:
I'll do this in separate PR.